lib/puppet/face/certificate.rb

```diff
@@ -53,9 +53,9 @@
 
     # Duplicate the option here explicitly to distinguish if it was passed arg
     # us vs. set in the config file.
-    option "--dns-alt-names "+ _("NAMES") do
-      summary _("Additional DNS names to add to the certificate request")
-      description Puppet.settings.setting(:dns_alt_names).desc
+    option "--subject-alt-names "+ _("NAMES") do
+      summary _("Additional subject alt names to add to the certificate request")
+      description Puppet.settings.setting(:subject_alt_names).desc
     end
```

Comment:
I think we may need to keep `--dns-alt-names` here in addition (with a deprecation warning) like we are doing elsewhere, to avoid breaking people.